libs/testing/src/perf/perf-setup.ts

```
@@ -0,0 +1,61 @@
+  const originalError = console.error;
+
+  beforeAll(() => {
+    console.log = jest.fn();
+    console.info = jest.fn();
+    console.debug = jest.fn();
+    console.warn = originalWarn;
+    console.error = originalError;
+  });
+
+  afterAll(() => {
+    console.log = console.log;
```

ESLint

[error] 'console.log' is assigned to itself.

(no-self-assign)